### ISSUE TITLE
Removing 'norm1' layer gives better result

### ISSUE THREAD

**Issue by Bonnie970** [toimcio/SegNet-tensorflow#9]
Hello, some posts suggest that lrn layer is not necessary when using batch norm. I found removing lrn layer gives 1-2% performance gain.
